docs/conf/proxy.md

```diff
@@ -12,20 +12,20 @@ Settings are loaded **in given order** and the **first existing value** is being
 
 Otherwise, proxy values can be specified manually when creating a backend:
 
-```scala
+```scala mdoc:compile-only
 import sttp.client._
 
-implicit val backend = HttpURLConnectionBackend(
-  options = SttpBackendOptions.httpProxy("some.host", 8080))
+implicit val backend = HttpURLConnectionBackend(options = SttpBackendOptions.httpProxy("some.host", 8080))
```

Comment:
I'd keep the linebreak so that the rendered code doesn't need to be side-scrolled